src/NuGetForUnity/Editor/PackagesConfigFile.cs

```diff
@@ -215,48 +215,57 @@ public void Save()
             contentIsSameAsInFilePath = filePath;
         }
 
+        /// <summary>
+        ///     Moves the packages.config file and its corresponding meta file to the given path. If there is no packages.config file on the current
+        ///     path, makes a default version of the file on the new path.
+        /// </summary>
+        /// <param name="newPath">Path to move packages.config file to.</param>
         internal static void Move(string newPath)
         {
             var oldFilePath = NugetHelper.NugetConfigFile.PackagesConfigFilePath;
             var oldPath = NugetHelper.NugetConfigFile.PackagesConfigPath;
             NugetHelper.NugetConfigFile.PackagesConfigPath = newPath;
-            if (!File.Exists(oldFilePath))
+            try
```

Comment:
Added try-catch in case there's any restriction for moving files (for example unauthorized access exception), in which case we log the error and set package config path to the old one.